browser-extensions/common/js/content-scripts/content-script-parkrunner.js

```diff
@@ -21,7 +21,7 @@ function get_table(id, caption) {
             return false
         }
         // Only include this table if the caption is correct
-        return table_captions[0].innerText == caption
+        return table_captions[0].innerText.replace('–', '').trim() == caption
```

Comment:
Why would they do that? But this seems like a sensible way to do it at the moment. It might break if the `-` isn't at the start, e.g. it's in the middle of a word, but it does look like it will work, lets see. Perhaps we should say "if starts with a `-`, remove that" to be sure.